wordpress-mobile/WordPress-Aztec-iOS
# Removes the EditableNode protocol.

<h3>Description</h3>

Removes the EditableNode protocol.  All nodes should have proper editing logic in place now.

Closes #307.
Fixes #319.
Fixes #325.

<h3>Testing:</h3>

**Test 1:**

Review the new unit test.  Run the unit tests.

**Test 2:**
1. Open the empty editor demo
2. Write "Hello World"
3. Press enter once
4. Type CMD + B
5. Try typing anything
Unfortunately we'll have to rely on unit tests for this one, as there's no simple scenario we can test against.

**Test 3:**

1. Open the empty editor.
2. Insert an image.
3. Edit text around the image.
4. See that it doesn't crash.
